packages/cardano-services/src/StakePool/HttpStakePoolMetadata/HttpStakePoolMetadataService.ts

```diff
@@ -62,12 +62,7 @@ export const createHttpStakePoolMetadataService = (
       throw error;
     }
   },
-
   async getStakePoolMetadata(hash, url) {
-    const errors: CustomError[] = [];
```

Comment:
This change here from returning an array of errors to a single error would be better as a separate commit, prior to making the other changes, but this is just for future reference, and no need to take action here.

Reply:
Yes, agreed.